playwright/e2e/atk_register_login/atk_register_login.spec.js

```diff
@@ -28,59 +20,37 @@ import userEtherealAccount from '../data/etherealUser.json';
 // Accounts module is enabled.
 import qaUserAccounts from '../data/qaUsers.json';
 
+// Set up Playwright.
```

Comment:
I was keeping these in the same order across all tests. Can you put it back to it's original location?